dts/xtensa/intel/intel_cavs.dtsi

```diff
@@ -51,5 +70,7 @@
 
 			status = "okay";
 		};
+
```

Comment:
Nitpick, but maybe remove the extra 2 breaks here and delete the break on line 49? I don't see these kind of line breaks in the other .dts files.